interface/client/templates/layout/sidebar.js

```diff
@@ -74,7 +74,7 @@ Template['layout_sidebar'].helpers({
         var template = Template.instance();
 
         if(this._id === 'browser') {
-            return DoogleLastVisitedPages.find({},{sort: {timestamp: -1}, limit: 25});
```

Comment:
Haha, I was doing some find-and-replace work and I thought I'd accidentally overwritten `Tabs`, especially because i kept getting an undefined error in the browser. Will fix.